obspy/signal/cross_correlation.py

```diff
@@ -103,9 +130,9 @@ def correlate(a, b, shift, demean=True, normalize=True, domain='freq'):
 
     .. note::
 
-        For most input parameters cross-correlation in frequency domain is much
+        For most input parameters cross-correlation using the FFT is much
         faster.
-        Only for small values of ``shift`` (``⪅100``) time domain
+        Only for small values of ``shift`` (``⪅100``) direct time domain
```

Comment:
I'm usually in favor of only using ASCII whenever possible.. that symbol is basically ``<=100``?

Reply:
Sure, will replace this fancy symbol.

Reply:
Ah, well, it is not exactly ``<=100``. It is approx. less than 100. We can also easily delete this note? New scipy has the `auto` method and users can look there.

Reply:
or just elaborate with words